waspc/src/Wasp/Util.hs

```diff
@@ -131,3 +146,25 @@ infixr 5 <:>
 
 (<:>) :: Monad m => m a -> m [a] -> m [a]
 (<:>) = liftM2 (:)
+
+type Checksum = Hex
+
+checksumFromString :: String -> Checksum
+checksumFromString = bytestringToHex . SHA256.hash . BSU.fromString
+
+checksumFromText :: Text -> Checksum
+checksumFromText = bytestringToHex . SHA256.hash . TextEncoding.encodeUtf8
+
+checksumFromByteString :: BSU.ByteString -> Checksum
+checksumFromByteString = bytestringToHex . SHA256.hash
+
+checksumFromFilePath :: FilePath -> IO Checksum
+checksumFromFilePath file = do
```

Comment:
You can turn this into a nice one-liner :)!

Reply:
Ah yeah, can go `checksumFromFilePath file = checksumFromByteString <$> B.readFile file` but I'll spend a few min thinking if I can get point free too :D

Reply:
Haha oh no you are becoming point-free person :D!
I can imagine doing `(checksumFromByteString <$>) . B.readFile` but that doesn't feel like much improvement I think.

Reply:
Haha nice, yeah I may just keep file in there but good one!